lib/Jit/options.cpp

```diff
@@ -48,9 +48,7 @@ void freeStringConfigValue(ICorJitInfo *CorInfo, UTF16CharT *Value) {
   return CorInfo->freeStringConfigValue((wchar_t *)Value);
 }
 
-Options::Options(LLILCJitContext *Context) : Context(Context) {}
-
-void Options::initialize() {
+JitOptions::JitOptions(LLILCJitContext &Context) : Context(Context) {
```

Comment:
Today no since all the setters are run within the Constructor.  This wasn't true in earlier implementations, and since we can imagine cases where some option could only be set sometime after the constructor is finished I left the reference as a member.  I'd be fine pulling it but it's a coin toss as to whether I wouldn't be adding it back next week.